app/assets/javascripts/admin/addon/routes/admin-backups-settings.js

```diff
@@ -0,0 +1,18 @@
+import Route from "@ember/routing/route";
+import { service } from "@ember/service";
+import SiteSetting from "admin/models/site-setting";
+
+export default class AdminBackupsSettingsRoute extends Route {
```

Comment:
Not sure if it's ideal to have to define a new route and controller for this for every config area, but not sure how else to do it right now. The only thing that will change for each of these routes is what arguments are passed to `SiteSetting.findAll`